internal/commands/inspect_image_test.go

```diff
@@ -215,6 +247,35 @@ func testInspectImageCommand(t *testing.T, when spec.G, it spec.S) {
   test.bp.two.local\s+2.0.0`)
 				})
 
+				it("displays process info for local and remote", func() {
+					h.AssertNil(t, command.Execute())
+					h.AssertContainsMatch(t, remoteOutput(outBuf), `Processes:
+  TYPE\s+SHELL\s+COMMAND\s+ARGS
```

Comment:
I think we should use regexes in test assertions sparingly. If there's any way to use a raw string (or perhaps a fixture), I think that route will be easier to maintain going forward.

Reply:
Due to the tabwriter, the amount of whitespace is unpredictable. We could get there with trial and error, but it would create a relatively fragile test in that if any of the inputs changed the header will also change. We'd also be essentially testing behaviour of the tabwriter. This way, we just care that their separated

Reply:
✅